src/odemis/driver/technolution.py

```diff
@@ -1467,7 +1463,8 @@ def _setCellDarkOffset(self, cellDarkOffset):
             raise ValueError("An incorrect shape of the dark offset parameters is provided.\n"
                              "Please change the shape of the dark offset parameters according to the shape of the mppc "
                              "detector.\n "
-                             "Dark offset parameters value remain unchanged.")
+                             "Dark offset parameters value remain unchanged."
+                             "%s, %s, %s", cellDarkOffset, len(cellDarkOffset), self._shape)
```

Comment:
I think a comment (+ enter) on what these output numbers are would be essential, otherwise it is even hard to keep these numbers apart since this will be a very big output so something like:
"Dark offset parameters value remain unchanged. dark offset values: %s \n length of the parameters %s and shape of these parameters saved %s", cellDarkOffset, len(cellDarkOffset), self._shape)

Note: that cellDarkOffset is quite a big variable (64 numbers) so the output error message will also be big/hard to read.

Reply:
A left over from my debugging. I remove it.